lib/new_relixir/plug/instrumentation.ex

```diff
@@ -45,8 +45,15 @@ defmodule NewRelixir.Plug.Instrumentation do
     model_name(model_type)
   end
 
-  defp infer_model(%Ecto.Changeset{model: model}) do
-    infer_model(model)
+  defp infer_model(%{model: model}), do: infer_model(model)
+
+  defp infer_model(%Ecto.Changeset{data: schema}) do
+    schema 
+      |> inspect 
+      |> String.split("{")
+      |> List.first
+      |> model_name
+      |> String.replace("\"", "")
```

Comment:
@veverkap It's too tricky and dirty way.
I think, `schema.__struct__ |> model_name` would be enough.